pymove/_version.py

```diff
@@ -419,8 +381,7 @@ def render_pep440_post(pieces):
 
 
 def render_pep440_old(pieces):
-    """
-    TAG[.postDISTANCE[.dev0]] .
+    """TAG[.postDISTANCE[.dev0]] .
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5497268)